src/hid-sensor.h

```diff
@@ -69,6 +71,8 @@ class hid_sensor : public raw_sensor_base
     std::vector< platform::hid_sensor > _hid_sensors;
     std::unique_ptr< frame_timestamp_reader > _hid_iio_timestamp_reader;
     std::unique_ptr< frame_timestamp_reader > _custom_hid_timestamp_reader;
+    //this map is intended to keep the values for gyro and accel sensitivity if the user changed them, available from FW >=5.16
```

Comment:
Rephrase "Keeps set sensitivity values for gyro and accel" no need to mention the FW version

Reply:
Done